transcom/mymove
# [ MB-16105 ] Display additional MTO Compliance info to office UI

## [Jira ticket](https://dp3.atlassian.net/browse/MB-16105)

## Summary

This PR displays the information required for MTO compliance in the Office UI. 

See [this figma link](https://www.figma.com/file/5AhuEkaH3bJVrtffU3wp22/Milmove-Office-App?type=design&node-id=11507-120468&mode=design&t=70uvHDDYnsj5Ekyj-0) for the design requirements.

Locally we are not using the real Contract number, so you will see a placeholder number if you run it on your machine.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the Office UI as a TOO or TIO
2. Go to a Move and click on the "Move task order" tab.
3. You should see the new information

### Frontend


### Backend


## Screenshots
New info in the header (now with 100% more right aligned Financial review links)
![image](https://github.com/transcom/mymove/assets/110134470/d6aa37db-928b-413e-b01a-c3ebad870421)

New info in the footer: 
![image](https://github.com/transcom/mymove/assets/110134470/d0ccd136-f043-4b49-9200-9bee70dd3374)
